### DISCUSSION TITLE
Release 23.04?

### DISCUSSION THREAD

**Discussion by josephholsten** [helix-editor/helix#6395] [Q&A]
Been a few months since last release. Are there any issues we want to close out before next release? Is 23.04 a reasonable goal?

While we’re at it, anyone got thoughts on a more concrete release schedule? Anything we should clarify in https://github.com/helix-editor/helix/wiki/FAQ#when-will-the-next-release-be?

**Comment by pascalkuthe** [comment]
The plan was originally to release in march but we might indeed miss that target.
There are still quite a few items left on the release milestone: https://github.com/helix-editor/helix/milestone/3
As stated in the FAQ release happen roughly every three months but there are no exact timelines. A release happens when we feel a good milestone has been reached. We are all volunteers working in our free time so giving exact estimates is not possible

**Reply by josephholsten** [reply]
The link is really useful, I’ll try to start testing PRs on that page. Thanks!

**Comment by archseer** [comment]
> Been a few months since last release. Are there any issues we want to close out before next release? Is 23.04 a reasonable goal?
> 
> While we’re at it, anyone got thoughts on a more concrete release schedule? Anything we should clarify in https://github.com/helix-editor/helix/wiki/FAQ#when-will-the-next-release-be?

My thoughts match https://github.com/BurntSushi/ripgrep/blob/master/FAQ.md#release

**Reply by josephholsten** [reply]
I love this. That’s actually one of the reasons I like the CalVer numbering, it feels more acceptable to just cut a release as a point-in-time than drive toward some arbitrary feature set or stability goal to tick over an arbitrary version number.
But, I also happily live off head on my primary machine.

**Comment by josephholsten** [comment]
If https://github.com/helix-editor/helix/milestone/3 is the authoritative list of what probably should be done for next release, then I’m happy closing.
